src/components/GordonLimitedAvailability/index.js

```diff
@@ -0,0 +1,40 @@
+import { Grid, Card, CardContent, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+const GordonLimitedAvailability = ({ pageName, backToLocation, backToLink, availableTo }) => {
+  let pageNameFormatted = pageName ? `The ${pageName} page` : 'This page';
+  return (
+    <Grid container justifyContent="center" spacing="16">
+      <Grid item xs={12} md={8}>
+        <Card>
+          <CardContent
+            style={{
+              margin: 'auto',
+              textAlign: 'center',
+            }}
+          >
+            <br />
+            <h1>{pageName + ' Unavailable'}</h1>
+            <h4>
+              {pageNameFormatted} is available for {availableTo} only.
```

Comment:
Since this is JSX, you can just put the ternary operator in this expression directly, rather than needing a variable for it.
```suggestion
  let pageNameFormatted = pageName ? `The ${pageName} page` : 'This page';
  return (
    <Grid container justifyContent="center" spacing="16">
      <Grid item xs={12} md={8}>
        <Card>
          <CardContent
            style={{
              margin: 'auto',
              textAlign: 'center',
            }}
          >
            <br />
            <h1>{pageName + ' Unavailable'}</h1>
            <h4>
              {pageName ? `The ${pageName} page` : 'This page'} is available for {availableTo} only.
```

Reply:
Eh, I liked it more taken out. I will make the changes though